Source/Settings.as

```diff
@@ -196,6 +208,41 @@ vec4 Setting_Gearbox_TextColor = vec4(1, 1, 1, 1);
 
 [Setting category="Gearbox" name="Font"]
 string Setting_Gearbox_Font = "DroidSans.ttf";
+
+
+[Setting category="Acceleration" name="Acceleration color" color]
+vec4 Setting_Acceleration_Positive_Color = vec4(0, 0.9f, 0, 1);
+
+[Setting category="Acceleration" name="Negative acceleration color" color]
+vec4 Setting_Acceleration_Negative_Color = vec4(0.8f, 0, 0, 1);
+
+[Setting category="Acceleration" name="Backdrop color" color]
+vec4 Setting_Acceleration_BackdropColor = vec4(0, 0, 0, 0.7f);
+
+[Setting category="Acceleration" name="Border color" color]
+vec4 Setting_Acceleration_BorderColor = vec4(1, 1, 1, 1);
+
+[Setting category="Acceleration" name="Border width" drag min=0 max=10]
+float Setting_Acceleration_BorderWidth = 3.0f;
+
+[Setting category="Acceleration" name="Border radius" drag min=0 max=50]
+float Setting_Acceleration_BorderRadius = 5.0f;
+
+[Setting category="Acceleration" name="Max Acceleration value" drag min=0 max=200]
```

Comment:
Capitalized "A" here is off, plus you could probably write out "Maximum", unless it's also written as "Max" in other Dashboard settings.